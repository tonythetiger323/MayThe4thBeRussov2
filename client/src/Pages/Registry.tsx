import React from 'react'
import RegistryCard from '../Components/RegistryCard'

const Registry = () => {
  return <RegistryCard/>
}

export default Registry;
