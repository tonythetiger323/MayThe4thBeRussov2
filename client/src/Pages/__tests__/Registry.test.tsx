import * as React from 'react'
import { shallow } from 'enzyme'
import Registry from '../Registry'

describe('Registry Page', () => {
  let wrapper: any
  beforeEach(() => (wrapper = shallow(<Registry />)))

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
