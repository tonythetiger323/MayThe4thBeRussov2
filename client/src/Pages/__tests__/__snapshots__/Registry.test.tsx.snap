// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registry Page should render correctly 1`] = `ShallowWrapper {}`;
